ui/components/app/custom-spending-cap/custom-spending-cap.js

```diff
@@ -139,6 +140,15 @@ export default function CustomSpendingCap({
     passTheErrorText(error);
   }, [error, passTheErrorText]);
 
+  useEffect(() => {
+    if (inputRef.current) {
+      document.getElementsByTagName('input')[0].focus({
```

Comment:
Maybe use `getElementById` and set an id on the FormField because this rely on element order which might change and break this code.

Alternatively, not sure why we can't just do `inputRef.current.focus`

Reply:
oh I left that by mistake, let me fix.